scripts/log.ps1

```diff
@@ -107,6 +137,50 @@ if ($IsLinux) {
         sudo apt-get install -y lttng-tools
         sudo apt-get install -y liblttng-ust-dev
     }
+    try { perf version | Out-Null}
+    catch {
+        Write-Host "Installing perf"
+        sudo apt-get install -y linux-tools-$(uname -r)
+        wget https://raw.githubusercontent.com/brendangregg/FlameGraph/master/stackcollapse-perf.pl -O /usr/bin/stackcollapse-perf.pl
+        wget https://raw.githubusercontent.com/brendangregg/FlameGraph/master/flamegraph.pl -O /usr/bin/flamegraph.pl
+    }
+}
+
+function Perf-Run {
+    if (!$IsLinux) {
+        # error
+    } else {
+        New-Item -Path $TempPerfDir -ItemType Directory -Force
+        $CommandSplit = $Command.Split(" ")
+        $OutFile = "server.perf.data"
+        if (!$Remote) {
+            $count = @(Get-ChildItem $TempPerfDir "client_*.perf.data").count
+            $OutFile = "client_$count.perf.data"
+        }
+        perf record -F 10 -a -g -o $(Join-Path $TempPerfDir $OutFile) $CommandSplit[0] $CommandSplit[1..$($CommandSplit.count-1)]
```

Comment:
The `-F 10` means only 10 times a second, right? That seems way too infrequent. Really, I think we want max.

Reply:
Sure, made lower for test purpose (too big and too slow for post processing)